src/Microsoft.PowerShell.Commands.Management/commands/management/WMIHelper.cs

```diff
@@ -162,7 +162,7 @@ internal override void StartOperation()
             }
```

Comment:
@SteveL-MSFT, your last commit had 1 failures in `PowerShell-CI-windows`
Set/New/Remove-Service cmdlet tests.Set-Service can change credentials of a service
```powershell
Expected a value, but got $null or empty.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Management\Set-Service.Tests.ps1: line 184
184:             $service | Should -Not -BeNullOrEmpty
```

Reply:
This is unrelated to this PR and a test issue with password complexity.